.github/workflows/sdk-release.yaml

```diff
@@ -11,21 +11,21 @@ jobs:
   generate-cdylibs:
     uses: IronCoreLabs/workflows/.github/workflows/rust-artifact.yaml@rust-artifact-v0
     with:
-      os_matrix: '["buildjet-2vcpu-ubuntu-2204", "buildjet-4vcpu-ubuntu-2204-arm", "macos-12", "macos-14"]'
+      os_matrix: '["buildjet-2vcpu-ubuntu-2004", "buildjet-4vcpu-ubuntu-2204-arm", "macos-12", "macos-14"]'
       build_profile: "release"
     secrets: inherit
 
   kotlin-release:
     needs: generate-cdylibs
-    runs-on: buildjet-2vcpu-ubuntu-2204
+    runs-on: buildjet-2vcpu-ubuntu-2004
     steps:
       - uses: actions/checkout@v4
       - uses: actions/download-artifact@v4
         with:
           path: ${{ github.workspace }}/kotlin/src/main/resources/
       - name: Rename directories
         run: |
-          mv buildjet-2vcpu-ubuntu-2204- linux-x86-64
+          mv buildjet-2vcpu-ubuntu-2004- linux-x86-64
```

Comment:
Are we doing this for Python as well?

Reply:
It looks like we are, is there some reason we shouldn't be? This was pulling Colt's ticket that was merged+closed to make the release use an older version of glibc, that never actually changed what was getting released. It was only changing CI.

Reply:
Sorry, I was trying to say that in this PR you're changing `generate-cdylibs` to generate for `2004` instead of `2204` (fixing what we missed previously) and then changing `kotlin-release` to release `2004` but haven't changed `python-release` below as well

Reply:
Oh, sure I can change it. It shouldn't matter either way as long as `generate-dylibs` runs on the version we want.

Reply:
> It shouldn't matter either way

Python's `mv` command will fail as it is right now since it doesn't match. The machine it's run on doesn't matter though